src/config/programming-languages.md

```diff
@@ -0,0 +1,14 @@
+# Building and Installing External Modules for Programming Languages
+
+To keep your system lean, Void splits packages and their devel packages. If you
+install program via pip, gem, etc. you may need to install the programming
+language's devel package to ensure your program builds.
+
+## Python
+
+To ensure python modules install correctly via pip, install the `python-devel`
```

Comment:
Also, I hope most Void users are using Python3 for now. The package for that one is `python3` c:

Reply:
Fixed